spec/models/accounting/qb/purchase_extractor_spec.rb

```diff
@@ -39,7 +40,24 @@
       expect(txn.account).to eq txn_acct
       expect(txn.amount).to equal_money(12345.67)
       expect(txn.sync_token).to eq "abc"
-      expect(txn.qb_object_subtype).to eq "Check"
+      expect(txn.disbursement_type).to eq "check"
+      expect { txn.save! }.not_to raise_error
```

Comment:
If we're just trying to test if the object is valid, a more performant and precise way of doing so is `expect(txn).to be_valid`. (More performance b/c it doesn't hit the DB.)

If we are concerned about e.g. a callback raising an error, this works, or we could just do `txn.save!` since an error would fail the spec anyway.